plan.md

```diff
@@ -1,9 +1,282 @@
-# Plan [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/phase-two/issues)
+# The Plan [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/phase-two/issues)
 
 
 ## Why?
 
 ### “_If you fail to plan, you are planning to fail!_” ~ <small>Benjamin Franklin</small>
 
-> This plan follows _logically_ on from the **Goal**,
-so if you have not read it please do that _first_!
+
+## What?
+
+If you've been around here for a while, you may have noticed that dwyl does many things:
+we build open source [apps](https://github.com/dwyl/time), websites and [modules](https://github.com/dwyl/hapi-auth-jwt2),
+we build them for ourselves and for clients, we write tutorials, promote open source,
+provide technology training, get involved in accessibility and women in tech initiatives.
+
+It didn't start out like this.
+
+We began with [a product MVP](https://github.com/dwyl/time),
+then in order to support [Founders & Coders](foundersandcoders.com) and ensure the lights
+could be kept on at this organisation that dwyl's co-founders were heavily involved
+in building, we [started to take on client projects](https://github.com/dwyl/ateam-proposal)(again)
+whilst building our learning community by sharing the knowledge as we went.
+
+This plan lays out **how dwyl aims to achieve its goals**, _clarifying_ the many hats
+dwyl wears and how they _work together_.
+
+## Contents Guide
+For the time being, we are putting everything into this one .md file, to be split
+out once reviewed from its rough draft form. We'll need a way to navigate!
++ [Context](#context)
++ [Branches](#branches)
```

Comment:
I don't think we _need_ to describe (_all_) five arms right now... the four @iteles has defined:
1. Product: Time
2. Service: ATM Workshops & Web App Development
3. Training: Web Skills
4. Co-Working: _QuietSpace_

are a good start. and `if` we are _successful_ with one or more of our existing "branches" (_or "arms"_), we can always extend later once "core" biz is _cashflow positive_.

also, the _beauty_ of Starfish is that we they can have more than five arms: 
![image](https://cloud.githubusercontent.com/assets/194400/23911716/8a697156-08d5-11e7-9db6-9fa9fd756a64.png)

and, a Starfish gives us a "cute" mascot:
![image](https://cloud.githubusercontent.com/assets/194400/23911859/fbbfad52-08d5-11e7-9756-f068ef3a8c21.png)
https://www.etsy.com/uk/listing/470894961/starfish-hair-clip-under-the-sea-party 😉